relay-server/src/actors/store.rs

```diff
@@ -914,8 +974,9 @@ enum KafkaMessage {
     Metric(MetricKafkaMessage),
     Profile(ProfileKafkaMessage),
     ReplayEvent(ReplayEventKafkaMessage),
-    ReplayRecording(ReplayRecordingKafkaMessage),
-    ReplayRecordingChunk(ReplayRecordingChunkKafkaMessage),
+    ReplayRecordingNotChunked(ReplayRecordingKafkaMessage),
+    ReplayRecording(ChunkedReplayRecordingKafkaMessage),
```

Comment:
Personally I would flip the variant names to match the struct names, e.g. `ReplayRecording(...)`, `ChunkedReplayRecording(...)`.

Reply:
I initially had the naming reversed.  Unfortunately, it translates the types to snake case and the types are flipped on the consumer because of legacy reasons.

Reply:
Let me think about a migration strategy.  Could be worth it.

Reply:
I misread this earlier I think.  I've updated the naming to be more consistent and I don't think I will migrate the message types.